### PR TITLE
Add pkgconfig recipe

### PR DESCRIPTION
**Pull request by lbdreyer**
Not to be confused with pkg-config!

This is a [build dependency of h5py](https://github.com/conda-forge/h5py-feedstock/blob/master/recipe/meta.yaml#L31).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This repo is a holding area for recipes destined for a conda-forge feedstock rep
 ## Getting started
 
 1. Fork this repository.
-2. Make a new folder in `recipes` for your package. Look at the example recipe and our [FAQ](https://github.com/conda-forge/staged-recipes/tree/readme_faq#faq) for help.
```
Comment (ocefpaf) [inline README.md]: :+1:

Comment (jakirkham) [inline README.md]: This change is fine, but I would like it to be a separate PR from the recipe addition as it is good git hygiene.

File: recipes/pkgconfig/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+about:
+  home: https://github.com/matze/pkgconfig
+  license: MIT
+  license_file: LICENSE
+  summary: 'A Python interface to the pkg-config command line tool'
+
+extra:
+  recipe-maintainers:
+    - lbdreyer
```
Comment (jakirkham) [inline recipes/pkgconfig/meta.yaml]: Feel free to add me. 😄

File: recipes/pkgconfig/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - pkgconfig
+
+about:
+  home: https://github.com/matze/pkgconfig
+  license: MIT
```
Comment (jakirkham) [inline recipes/pkgconfig/meta.yaml]: Redundant and optional, but would be nice to have `license_family: MIT`.

File: recipes/pkgconfig/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{% set name = "pkgconfig" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "4de443a62f4b83d883a7b3895d4d4d3ca1934383ebdbeb0f2e6ab126085d812a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/pkgconfig/meta.yaml]: Would suggest this be `{{ name }}-{{ version }}.tar.gz`.

Note: It doesn't matter what the actual file downloaded is named as `conda-build` will rename it with the `fn` provided. Doing it as proposed will be helpful for people building offline, but does not strictly matter for conda-forge.

File: recipes/pkgconfig/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose >=1.0
```
Comment (jakirkham) [inline recipes/pkgconfig/meta.yaml]: Note: Needed for the [build]( https://github.com/matze/pkgconfig/blob/v1.1.0/setup.py#L15 ).

File: recipes/pkgconfig/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
```
Comment (jakirkham) [inline recipes/pkgconfig/meta.yaml]: Should we add `pkg-config` so this works more easily out of the box?

**Comment by jakirkham** [comment]
Please see PR ( https://github.com/conda-forge/pkgconfig-feedstock/pull/1 ) for these changes.
